Apps/Examples/Examples/All Examples/FlyToExample.swift

```diff
@@ -42,11 +42,8 @@ public class FlyToExample: UIViewController, ExampleProtocol {
                                 pitch: 50)
 
         mapView.cameraManager.setCamera(to: start) { _ in
-            self.mapView.cameraManager.fly(to: end) { [weak self] _ in
-                print("Camera fly-to finished")
-                // The below line is used for internal testing purposes only.
-                self?.finish()
```

Comment:
`self.finish` needs to be called in the completion handler for the flyTo below -- this is probably the cause of the failing tests.